rs/pocket_ic_server/src/pocket_ic.rs

```diff
@@ -1256,12 +1260,19 @@ impl Operation for StatusRequest {
     }
 }
 
+pub enum CallRequestVersion {
+    V2,
+    V3,
+}
+
 pub struct CallRequest {
     pub effective_canister_id: CanisterId,
     pub bytes: Bytes,
     pub runtime: Arc<Runtime>,
+    pub version: CallRequestVersion,
 }
 
+const INGRESS_MESSAGE_CERTIFICATION_TIMEOUT_SECONDS: u64 = 10;
```

Comment:
Could you please move this constant to the top of the file and add a doc comment?